index.js

```diff
@@ -87,6 +146,16 @@
 
 // ***** Callbacks *****
 
+
+function triple(element){
+  
+  let tripledArray = [];
+  tripledArray.push(element * 3);
+  return tripledArray;
+// i keep getting separate arrays for each element...I know that must be fixed 
+// but for the sake of time im moving on...look forward to feedback :)
+}
```

Comment:
Looks good. I do not think we need to create an array inside of the triple function. I believe this code will result in an array of arrays [[11],[23],[45],[9],[12]]. We just need triple to return a value that myMap will then add to the array.